### ISSUE TITLE
Wrong GPS time

### ISSUE THREAD

**Issue by slavka93** [freshollie/UsbGps4Droid#23]
Hello. I live in Russia, Moscow region. I want to express my deep gratitude for your application. Everything works great. Only after turning on the GPS item, the time is set to +12 hours and the region +3 East Africa. You have the opportunity to edit this time adjustment in the application, otherwise in the navigation programs the themes of the night version are installed during the day. Thank.

**Comment by slavka93** [comment]
UsbGps4Droid-v2.2.0-release.apk

**Comment by freshollie** [comment]
Hi. Are you using the gps time to set the system time? Is that causing the issue?

**Comment by slavka93** [comment]
Yes

**Comment by freshollie** [comment]
Think I've found the issue, but I don't have the hardware with me to check. I will make the change, and compile an APK which you can test here.

**Comment by slavka93** [comment]
Yes, of course, I will test everything and write about the results.

**Comment by slavka93** [comment]
Hello. Now I noticed that the program took 12 hours from the actual time. It will take 12 hours, then it will add. It happens that the time does not change and it corresponds to the actual one.

**Comment by freshollie** [comment]
Hey, sorry I am a bit confused. You might be able to explain better with screenshots :)

**Comment by freshollie** [comment]
Also, can I confirm are you on a GMT+12 timezone?

**Comment by slavka93** [comment]
I am in time zone +3. Photos will now be.

**Comment by slavka93** [comment]
![IMG_20200912_143701](https://user-images.githubusercontent.com/71120654/92994848-c4e25e80-f506-11ea-8330-1b49e7214bcf.jpg)
In this photo, the top line is the actual time. The bottom line is -12 o'clock.

**Comment by slavka93** [comment]
![IMG_20200912_143457](https://user-images.githubusercontent.com/71120654/92994924-489c4b00-f507-11ea-916e-62368d818b52.jpg)
In this photo, the top line is the actual time. The bottom line is +12 o'clock.

**Comment by slavka93** [comment]
And the date is always wrong, but it's not important to me.

**Comment by freshollie** [comment]
Can you try this version? [UsbGps4Droid-v2.2.0-release.apk.zip](https://github.com/freshollie/UsbGps4Droid/files/5212635/UsbGps4Droid-v2.2.0-release.apk.zip)

**Comment by slavka93** [comment]
Yes now.

**Comment by slavka93** [comment]
Thank!!!! It works !!!! Out of 5 system starts, 5 correct readings.

**Comment by slavka93** [comment]
Please tell me. Is it possible to run the program in the background? Android 9.

**Comment by slavka93** [comment]
By adjusting the time, I'll go for a ride, I'll see.

**Comment by freshollie** [comment]
Not sure on your issue with the background, can you open another issue to track? I'm gonna close this as I've published v2.2.1 with the fix.

**Comment by slavka93** [comment]
Everything works great! Many thanks!
